h/assets.yaml

```diff
@@ -100,10 +100,14 @@ config:
   output: scripts/config.min.js
   filters: uglifyjs
   contents:
-    - output: scripts/config.js
+    - output: scripts/config.accounts.js
       filters: browserify
       contents:
         - h:static/scripts/config/accounts.coffee
+    - output: scripts/config.via.js
```

Comment:
I would just add this content to the bundle above. It'll mean it's all in one JS file even for development, but source maps should work (at least in Chrome, FF has issues I won't get into here), but it also means you don't need to add a `rm` to `make clean` for config.accounts.js.